integration_tests/features/environment.py

```diff
@@ -21,3 +24,9 @@ def after_scenario(context, scenario):
         print_exception(*context.exc)
         _etype, exception, _tb = context.exc
         raise exception
+
+
+def before_tag(context, tag):
```

Comment:
ATL: Where do we use this function?

Reply:
It's used by behave: https://behave.readthedocs.io/en/stable/api.html?highlight=before_tag#environment-file-functions

Reply:
thanks!